src/makeCalculator.js

```diff
@@ -36,8 +36,47 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, value) {
+      return callback.call(this, value);
```

Comment:
You may just call operation and then return `this` here. It helps you avoid code duplicate in each operation (you returned `this`)